pkg/generated/openapi/def.bzl

```diff
@@ -14,17 +15,17 @@ def openapi_library(name, tags, srcs, openapi_targets=[], vendor_targets=[]):
   )
   go_genrule(
       name = "zz_generated.openapi",
-      srcs = srcs + ["//hack/boilerplate:boilerplate.go.txt"],
+      srcs = srcs + ["//" + openapi_pkg_prefix + "hack/boilerplate:boilerplate.go.txt"],
```

Comment:
Could you depend on the boilerplate file in https://github.com/kubernetes/repo-infra/tree/master/verify/boilerplate?

It might make sense to move this entire rule to repo-infra.

Reply:
I think that should probably fall to the maintainers of repo-infra.  I'm just trying to get federation out of tree.